srcpkgs/vdf/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'vdf'
+pkgname=vdf
+version=3.2
+revision=1
+build_style=python-module
```

Comment:
There's no use in adding a version for Python 2 while it is on its way out. Please make the package python3 only and rename it to `python3-vdf` according to the naming convention for python libraries.

Reply:
fixed. Since the pkgname is now python3-vdf I have to set the wrksrc manually. Is that expected or should I change something?

Reply:
That's expected, we do it for most python modules.